### PR TITLE
Bug 1303402, Update style of "Using Jenkins as a Source-To-Image builder"; s/JSON/YAML/

### PR DESCRIPTION
**Pull request by tnguyen-rh**
This is a followup of https://github.com/openshift/openshift-docs/pull/1316 (and thus inherits its labels and milestone).  No tech review necessary.

@adellape PTAL

### CODE REVIEW

File: using_images/other_images/jenkins.adoc [hunk 1]
```diff
@@ -119,64 +119,67 @@ Once selected, you must link:../../dev_guide/templates.html#creating-resources-f
...
 to copy your custom Jenkins Jobs definitions, additional
-plugins or replace the provided `config.xml` file with your own, custom, configuration.
+plugins or replace the provided *_config.xml_* file with your own, custom, configuration.
 
 In order to include your modifications in the Jenkins image, you need to have a Git
-repository, with the following directory structure:
+repository with the following directory structure:
+
+*_plugins_*::
+This directory contains binary Jenkins plugins you want to copy into Jenkins.
```
Comment (bfallonf) [inline using_images/other_images/jenkins.adoc]: Missing a "the"?

"This directory contains the binary Jenkins plugins you want to copy into Jenkins"

File: using_images/other_images/jenkins.adoc [hunk 2]
```diff
@@ -119,64 +119,67 @@ Once selected, you must link:../../dev_guide/templates.html#creating-resources-f
...
+This file contains your custom Jenkins configuration.
 
-Note that the content of `configuration` folder will be copied into the `/var/lib/jenkins`
-folder, so you can also include additional files (like `credentials.xml`, etc.).
+The contents of the *_configuration/_* directory will be copied
+into the *_/var/lib/jenkins/_* directory, so you can also include
+additional files, such as *_credentials.xml_*, there.
 
-For example, you can use following build configuration to customize the Jenkins
+Following is an example build configuration that customizes the Jenkins
```
Comment (bfallonf) [inline using_images/other_images/jenkins.adoc]: "The following..."

**Comment by bfallonf** [comment]
@tnguyen-rh A couple nits, but otherwise LGTM!

File: using_images/other_images/jenkins.adoc (hunk 1 above)
Comment (tnguyen-rh) [inline using_images/other_images/jenkins.adoc]: @bfallonf Good catch, thanks.  After some pondering i think the most accurate word in that position would be "those":

"This directory contains those binary Jenkins plugins you want to copy into Jenkins."

**Comment by tnguyen-rh** [comment]
@bfallonf 
Thanks for the review.  I've addressed both points.

[rev_history]
|link:../using_images/other_images/jenkins.html[Other Images -> Jenkins]

|Added information on how to link:../using_images/other_images/jenkins.html#jenkins-as-s2i-builder[use Jenkins as a source-to-image builder].
%

**Pull request merged**